README.md

```diff
@@ -30,7 +30,7 @@ Learn more about BlenderNC in the official documentation at [https://blendernc.r
 Contributing
 ------------
 
-All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome. More information about contributing to BlenderNC can be found at our [Contribution page](https://blendernc.readthedocs.io/contribute).
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome. More information about contributing to BlenderNC can be found at our [Contribution page](https://blendernc.readthedocs.io/en/latest/contribute.html).
```

Comment:
We could use an in repo relative like like [Contribution page](docs/...) as well, [Github](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) also recommends a `contributing.md` which is also an option.

Reply:
Could you clarify the contribution page part? I can follow creating a contributing.md, but what do you mean with " repo relative like Contribution page".

Reply:
Fixed the typo that had made it incomprehensible.

Reply:
Sorry, I still don't get which typo?